### ISSUE TITLE
Splitted Archives for RAW-Backup

### ISSUE THREAD

**Issue by BenjaminBeichler** [digint/btrbk#125]
Hi,

I would like to storage my backup on Amazon Cloud Drive, but inherently they have a max file size of 50GB. Since such caps could also apply for other storages (I could imagine old FAT32 drives, but that seems too legacy), it would be nice, if btrbk could self handle splitting up the raw files. 

I know another workaround could be multiple loop-back devices connected to a btrfs big storage pool, but unfortunately especially acd_cli could not handle random writes to files.

**Comment by digint** [comment]
I implemented a quick'n'dirty `raw_target_split` support in [raw_target_split branch](/digint/btrbk/tree/raw_target_split), commit a107b722348b05e1f0ed8b273b59da41ca025dfa.

If you set `raw_target_split <nn>[K,M,G,...]` in the config file, btrbk will pipe the raw target files through the "split" command, resulting in `<filename>.split.aa`, `<filename>.split.ab`, ...

example:
```
[...]
raw_target_split  50G

volume /mnt/btr_pool
  subvolume home
    target raw ssh://cloud.example.com/backup
```

Please comment here if you have some suggestions about this.
(I'm in vacation next week, so don't expect any answers then...)

**Comment by digint** [comment]
Removed branch raw_target_split, this has been implemented in 28c65e46759a6a6866d3ae5a52ed162f028cb4d4
